Cargo.toml

```diff
@@ -24,7 +24,7 @@ log-impl = []
 [target.'cfg(windows)'.dependencies]
 sapp-windows = { path ="./native/sapp-windows", version = "=0.2.19" }
 
-[target.'cfg(target_os = "linux")'.dependencies]
+[target.'cfg(all(unix, not(any(target_os = "macos", target_os = "ios", target_os = "android"))))'.dependencies]
```

Comment:
Hi! I liked the `any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd"))'.` one from your `quad-snd` a bit more - it assumes someone actually tested it on said OSes and the intention is clear, while a long line with `not` is a bit less self-explanatory. Any good reasons not to use it here?